api/app/src/main/kotlin/packit/repository/PacketGroupDisplayRepository.kt

```diff
@@ -0,0 +1,11 @@
+package packit.repository
+
+import org.springframework.security.access.prepost.PostFilter
+import packit.model.PacketGroupDisplay
+
+interface PacketGroupDisplayRepository : ViewRepository<PacketGroupDisplay, String> {
```

Comment:
Spring works out from the name of the T type you pass it what view it needs to query from?

Reply:
I think instead it reads the slightly-misnamed 'Table' annotation that lives on PacketGroupDisplay.